doc/cascadia/profiles.schema.json

```diff
@@ -753,7 +753,17 @@
         },
         "backgroundImage": {
           "description": "Sets the file location of the image to draw over the window background.",
-          "type": ["string", "null"]
+          "oneOf": [
+            {
+              "type": "string"
+            },
+            {
+              "enum": [
+                "desktopWallpaper"
+              ],
+              "type": "null"
```

Comment:
i don't believe this to be correct? it cannot be both an enum value and a null

Reply:
I think this is right now?